integration/test/pom.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>beadledom-integration-test</artifactId>
+
+  <parent>
+    <artifactId>beadledom-integration</artifactId>
+    <groupId>com.cerner.beadledom</groupId>
+    <version>3.3-SNAPSHOT</version>
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.cerner.beadledom</groupId>
+      <artifactId>beadledom-integration-client</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.cerner.beadledom</groupId>
+      <artifactId>beadledom-integration-api</artifactId>
+    </dependency>
```

Comment:
super nit: order this so that api comes first before client dependency

Reply:
Re-ordered: https://github.com/cerner/beadledom/pull/149/commits/0ca69d4b1c84c077044606638b2b8f9a550cf38b